Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
The desire to let designers use the Inspector to set up Events this way is great for doing quick prototyping, but isn't necessarily the best design practice.  I've already discussed separating this out as a new PR and trying to get that same functionality (assigning events in Inspector) without using Unity Events. 

For reference:
http://jacksondunstan.com/articles/3335